### ISSUE TITLE
Logs don't work when in AP mode

### ISSUE THREAD

**Issue by piv3rt** [rstrouse/ESPSomfy-RTS#325]
### Hardware

ESP32

### Firmware version

v2.4.1

### Application version

v2.4.1

### What happened? What did you expect to happen?

When first booting the ESP32, when it's still in access point mode and broadcasting the "ESPSomfy RTS" SSID, receiving logs doesn't work.

This could be useful when testing on a remote location with no internet router (new house for example).

### How to reproduce it (step by step)

```bash
1. Boot the ESP32
2. Connect to WiFi "ESPSomfy RTS"
3. Configure and enable radio
4. Go to Radio > Logs to listen to signals
5. No signal is logged when pressing buttons on an existing remote
```

**Comment by rstrouse** [comment]
I will take a look at this possibility but the initial AP mode does not start up the radio logs.  The software is not designed to operate without some sort of a network connection.

**Comment by rstrouse** [comment]
In the next release (v2.4.3) the radio will process in AP mode if it is initialized.

**Comment by rstrouse** [comment]
I am going to close this for now since this function is available in v2.4.3.  I should be releasing this over the weekend.
